### ISSUE TITLE
Why there are intermediate kernels for x4?

### ISSUE THREAD

**Issue by JingyunLiang** [sefibk/KernelGAN#52]
In the given DIVRK dataset, there are intermediate kernels for x4. I guess they are x2 kernels and the given x4 kernels are derived from intermediate x2 kernels. Then, the derived x4 kernels are used to generate LR image. Am I right?

Why don't directly generate x4 kernels and use them to blur HR? What's the difference between anisotropic Gaussian x4 kernel  and derived x4 kernel (from anisotropic Gaussian x2 kernel)? Would it be fair to compare with other methods that are using anisotropic Gaussian x4 kernels?

**Comment by sefibk** [comment]
I am not sure I understand the second part.
The first part - correct

**Comment by Zj-BinXia** [comment]
Yes, I have the same question. Compared with generating the  x4 kernels anisotropic Gaussian based on x2 kernels anisotropic Gaussian kernels, why can't generate the  x4 kernels anisotropic Gaussian directly?

**Comment by sefibk** [comment]
You can do that as well.
As I recall (it was a while ago), we spoke with a physics/vision/optics specialistic who claimed that more realistic and natural kernels are a multiplication of smaller scale factors than a big one that can't be represented as 2 smaller ones
